spec/fixtures/static_pages/mp.php?mpn=Christine_Milne&mpc=Tasmania&house=senate&display=everyvote.html

```diff
@@ -106,14 +106,14 @@ <h2 class="social-share-heading small">
 
 <h2 id="divisions">All Votes</h2>
 <p>All votes Senator Christine Milne could have attended.</p>
-<ol class="display-house-all divisions-list list-unstyled">
+<ol class="display-house-senate divisions-list list-unstyled">
```

Comment:
Okay, I think I figured out why. I believe I fixed a bug that raised another one. In the `_divisions.html.haml` there is a bit of code like this: `%ol.divisions-list.list-unstyled{class: active_house_for_list_class(@house)}`. The method `active_house_for_list_class` is defined in `divisions_helper.rb`. The method is as follows:

```ruby
def active_house_for_list_class(house)
    if house == "representatives"
      "display-house-representatives"
    elsif house == "senate"
      "display-house-senate"
    elsif house == nil
      "display-house-all"
    end
end
```

The bug was the `@house` variable was always `nil`. And therefore only the class `display-house-all` was displayed. But I accidentally fixed that, which changed the HTML which I think is ok. However, there are no CSS classes defined for `display-house-senate` and `display-house-representatives`. There is only a class for `display-house-all` defined at `_division-list.scss`.  

After all explanation, my questions is: should I create the missing CSS classes with the same style as  `display-house-all`?